pfb-analysis/connectivity/reachable_roads_high_stress_calc.sql

```diff
@@ -21,7 +22,7 @@ FROM    neighborhood_ways r1,
                     link_cost AS cost
             FROM    neighborhood_ways_net_link',
             v1.vert_id,
-            10560,
+            :nb_boundary_buffer,
```

Comment:
This suggestion is essentially the goal of #81. 

@spencerrecneps if we dropped the boundary buffer to just be 1/2 of max trip distance, would that be a sensible optimization from your side? Currently the boundary buffer == max trip distance, which is more than necessary.